### PR TITLE
Driveby refactor

### PR DESCRIPTION
**Pull request by robinjam**
[Trello card](https://trello.com/c/pOBu4Y90/3452-change-govuk-dependabot-merger-api-5)

Some refactors that I've pulled out of a different PR I'm working on, so that I can get feedback on the changes early.

### CODE REVIEW

**Comment by robinjam** [comment]
Forgot to run the linter locally, will fix the issues now

File: lib/date_ext.rb
Comment (ChrisBAshton) [inline lib/date_ext.rb:1]: Would `date_monkeypatch.rb` be a more appropriate name?
(Nice refactor btw!)

**Review by ChrisBAshton — APPROVED** [review]
LGTM! ⭐

File: lib/date_ext.rb
Comment (robinjam) [reply lib/date_ext.rb:1]: I should probably just call it `bank_holidays.rb` actually, my bad!

**Comment by robinjam** [comment]
Thanks @ChrisBAshton!

**Pull request merged**